src/StackAdmin/Azs.Subscriptions/generate.ps1

```diff
@@ -1,6 +1,6 @@
 
-$rpName = "subscriptions"
-$name = "Subscriptions"
+$rpName = "subscription"
+$name = "Subscription"
 $location = Get-Location
 $psswagger = "D:\github\PSswagger"
```

Comment:
We don't want this hard-coed path, right?

Reply:
yes, we don't want the hardcoded path, the script needs to change to clone or download the psswagger instead of taking its path.. I will file an issue and address it later